src/encoded/authentication.py

```diff
@@ -279,19 +340,41 @@ def logout(context, request):
     The front-end handles logging out by discarding the locally-held JWT from
     browser cookies and re-requesting the current 4DN URL.
     """
-    #request.session.invalidate()
-    #request.response.headerlist.extend(forget(request))
+    request_parts = urlparse(request.referrer)
+    request_domain = request_parts.hostname
+
+    # Deletes the cookie
+    request.response.set_cookie(
+        name='jwtToken',
+        value=None,
+        domain=request_domain,
+        max_age=0,
+        path='/',
+        overwrite=True
+    )
+
+    request.response.status_code = 401
+    request.response.headers['WWW-Authenticate'] = (
+        "Bearer realm=\"{}\", title=\"Session Expired\"; Basic realm=\"{}\""
+        .format(request.domain, request.domain)
+    )
+
+    return { "deleted_cookie" : True }
```

Comment:
This is the result of the login endpoint?

Reply:
The /logout endpoint